Gemfile

```diff
@@ -28,7 +28,7 @@ gem 'browser'
 gem 'charlock_holmes', '~> 0.7.5'
 gem 'iso-639'
 gem 'cld3', '~> 3.2.0'
-gem 'devise', '~> 4.2'
+gem 'devise', git: 'https://github.com/plataformatec/devise.git', ref: '1009096172f2cbc86bcd54d053c89a09be67fb9f'
```

Comment:
https://github.com/plataformatec/devise/issues/4719

It has not been released yet.